packages/sitecore-jss/src/graphql/graphql.test.ts

```diff
@@ -44,7 +44,7 @@ describe('graphql', () => {
             },
           });
 
-        const result = await getAppRootId(client, 'siteName', 'language');
+        const result = await getAppRootId(client, 'siteName', 'en');
```

Comment:
Seems like we should have 2 new tests which are variants of these existing "should return" ones.
1. 'valid app root ID' (existing - mocks a "hit" for the passed 'language')
2. 'valid app root ID with en fallback' (new - mocks a "miss" for the passed 'language' but "hit" for 'en')
3. 'null if no app root found' (existing - mocks a "miss" for 'en')
4. 'null if no app root found with en fallback' (new - mocks a "miss" for the passed 'language' and "miss" for 'en')